examples/dump.rs

```diff
@@ -24,15 +24,12 @@ use wasmbin::Module;
 #[derive(Subcommand)]
 enum DumpSection {
     All,
-    Custom {
-        name: String,
-    },
+    Custom { name: String },
```

Comment:
Just to be sure - is this produced by newer rustfmt? I assume you ran `cargo fmt` over everything, just double-checking as I see I haven't added cargo fmt check to the CI.